src/task10/task10.js

```diff
@@ -0,0 +1,21 @@
+var obj = JSON.parse(ANCESTRY_FILE)
+
+function average(array) {
+    return array.reduce((a, b) => a + b) / array.length;
+}
+
+function mothersAvergeAge(array) {
```

Comment:
What matter in this topic is to learn hoks (Higher-Order Functions), so use them.
And there also should be **byName** helper function.